src/components/PopupWithForm.js

```diff
@@ -32,9 +32,26 @@ export default class PopupWithForm extends Popup {
     this._popupForm.addEventListener("submit", (e) => {
       e.preventDefault();
       this._handleFormSubmit(this._getInputValues());
-      this.close();
+      //this.close();
     });
     //The setEventListeners() method of the PopupWithForm class should add a submit event listener to the form
     //and call the setEventListeners() method of the parent class
   }
+
+  renderLoadingSave(loading) {
+    const submitButton = this._popupForm.querySelector(".modal__button");
+    if (loading) {
+      submitButton.textContent = "Saving...";
+    } else {
+      submitButton.textContent = "Save";
+    }
+  }
+  renderLoadingCreate(loading) {
```

Comment:
`renderLoadingCreate` is not needed in `PopupWithForm` because it's a duplication of code and some hardcode for text `Create`

Please, make universal `renderLoading` with 2 params

```js

renderLoading(isLoading, loadingText='Saving...') {
```